### ISSUE TITLE
Feature Request: getblockchaininfo should return info about the overall transparent value pool

### ISSUE THREAD

**Issue by daira** [zcash/zcash#4294]
The `getblockchaininfo` RPC output currently has a "valuePools" key that gives information about the total value in the Sprout and Sapling value pools:
```
  "valuePools": [
    {
      "id": "sprout",
      "monitored": true,
      "chainValue": 142961.23820218,
      "chainValueZat": 14296123820218
    },
    {
      "id": "sapling",
      "monitored": true,
      "chainValue": 258439.66179030,
      "chainValueZat": 25843966179030
    }
  ],
```

These are independently computed by the node (see #2795 for when this was originally added for Sprout). I propose adding a "transparent" value pool computed in a similar way, and with a similar fallback mechanism for when the node has not been synced from scratch (see #3885).

This is partly motivated by the observation that several third-party sites are incorrectly calculating the total issuance after Blossom (because they are computing it based on the block height and the pre-Blossom issuance rules).

**Comment by daira** [comment]
@str4d says:
> you can inspect the transparent value pool by looking at `gettxoutsetinfo`

@daira:
> oh cool
> that seems to take a few seconds to run. is it summing up the utxo values on the fly?!

@sean:
> it might be adding up the data from the block index

**Comment by str4d** [comment]
The limitation of `gettxoutsetinfo` is that it only works for the current block height. That is fine for `getblockchaininfo`, but our shielded pool totals are stored per-block, sand also used in `getblock`. It would be useful to have transparent totals in the latter as well.

**Comment by nuttycom** [comment]
Fixed in #4470
